components/AddressSvmTable.tsx

```diff
@@ -0,0 +1,51 @@
+import { StyledTd } from "./Table";
+
+const AddressSvmTable = ({
+  entries,
+  explorer,
+}: {
+  entries: { name: string; value: string }[];
+  explorer: string;
+}) => {
+  return (
+    <table>
+      <tbody>
+        {entries.map(({ name, value }) => {
+          const component = (
+            <code
+              className={
+                "nx-border-black nx-border-opacity-[0.04] nx-bg-opacity-[0.03] nx-bg-black nx-break-words nx-rounded-md nx-border nx-py-0.5 nx-px-[.25em] nx-text-[.9em] dark:nx-border-white/10 dark:nx-bg-white/10  "
+              }
+            >
+              {value.startsWith("PK-") ? value.slice(3) : value}
+            </code>
+          );
+          const addLink =
+            explorer.includes("$ADDRESS") && value.startsWith("PK-");
```

Comment:
this is a bit hacky.
Maybe change entries type to `{ name: string; value: string; addLink?: bool }`